ui/app/models/auth-config/azure.js

```diff
@@ -0,0 +1,38 @@
+import Ember from 'ember';
+import DS from 'ember-data';
+
+import AuthConfig from '../auth-config';
+import fieldToAttrs from 'vault/utils/field-to-attrs';
+
+const { attr } = DS;
+const { computed } = Ember;
+
+export default AuthConfig.extend({
+  tenantId: attr('string', {
+    label: 'Tenant ID',
+    helpText: 'The tenant id for the Azure Active Directory organization',
```

Comment:
Opinion on capitalizing ID? It's `id` in the API, and it's not like ID is a real acronym either, but `id` reads strangely in a regular sentence (comes up below too)

Reply:
Yep, sounds good - added in the latest!